packages/frontend/core/src/blocksuite/presets/ai/chat-panel/chat-cards.ts

```diff
@@ -443,41 +465,28 @@ export class ChatCards extends WithDisposable(LitElement) {
         if (hasImages) {
           card.images = images;
         }
-        this._updateCards(card);
+
+        if (this.cards.length === 0) {
+          this._updateCards(card);
+        } else {
+          this.cards.push(card);
```

Comment:
after looking into the logic of `_updateCards()`, it seems to be unnecessary for both condition.
in other word, `cards.push` and `requestUpdate` are ok for both branches